### PR TITLE
Initial version of liquidation workflow panels

### PR DESCRIPTION
**Pull request by theodubiel**
Added chartjs to project.
Moved Papi's CLI for anchor liquidations to a hook.
Created demo graph of total bids.
Created panel for placing bids (no functiaonlty yet)
Created panel for displaying current bids of the wallet connected (this works but needs formatting and more functionality).

### CODE REVIEW

File: src/components/free-willy/LiquidationBidChart.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+                </Stack>
+            )
+        } else {
+            // TODO: Add loading animation.
+            return (<></>)
+        }
+    }
+
+    return (
+        <>{renderGraph()}</>
```
Comment (vladjdk) [inline src/components/free-willy/LiquidationBidChart.tsx]: unneeded <></>

File: src/components/Wallet.tsx [hunk 2]
```diff
@@ -29,9 +29,8 @@ export function Wallet() {
   return (
     <div>
         <Chip label={getButtonText()} sx={{bgcolor: 'primary.light'}} onClick={handleClickOpen} />
-        <Dialog
+        {open && <Dialog
```
Comment (theodubiel) [inline src/components/Wallet.tsx:32]: Was there a bug with the original logic?

File: src/components/free-willy/LiquidationBidChart.tsx [hunk 3]
```diff
@@ -52,6 +52,7 @@ export default function LiquidationBidChart() {
 
     const renderGraph = () => {
         if (data) {
+            console.log(data);
```
Comment (theodubiel) [inline src/components/free-willy/LiquidationBidChart.tsx]: Remove console log

**Review by vladjdk — APPROVED** [review]
looks awesome

//approved

**Pull request merged**